### DISCUSSION TITLE
Opportunities in the report states they dont directly affect the performance score. So how do we improve the score actually

### DISCUSSION THREAD

**Discussion by SivasubramanianV** [GoogleChrome/lighthouse#15426] [Q&A] [answered]
<img width="903" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/68318570/8900f719-cae9-415b-ba30-1c08b5ecadaa">
 I see above image in the report and it states like this. How do we improve the score? I have to create some Jira tickets for UI developers to fix something so that it results in increased performance score. What should i use from report to increase the score?
Thanks

**Comment by connorjclark** [comment, marked as answer]
The opportunities listed map to work items.

The meaning is that they do not _directly_ calculate the score. They point out issues with the page, that if you fix, will improve one of the few metrics that do make up the score. They have an indirect impact, not direct.

**Reply by SivasubramanianV** [reply]
Thank you Connor.
